travis.yml

```diff
@@ -0,0 +1,30 @@
+language: php
+
+php:
+    - 7.2
+    - 7.3
+    - nightly
+
+env:
+    matrix:
+        -
```

Comment:
Not too experienced with travis, but is this supposed to be an empty item? Seems a bit out of place